settings/default.py

```diff
@@ -27,6 +27,29 @@
 DISCORD_WHITELIST_CHANNELS = env.json("DISCORD_WHITELIST_CHANNELS", "[]")
 DISCORD_SERVER_ID = env.int("DISCORD_SERVER_ID", 0)
 
-MESSAGE_REPLY = "0"
-REACTION = "1"
-POSTED_MESSAGE = "2"
+# Event Types
+MESSAGE_REPLY = env.int("MESSAGE_REPLY_TYPE_CODE", 0)
+REACTION = env.int("MESSAGE_REACTION_TYPE_CODE", 1)
+POSTED_MESSAGE = env.int("MESSAGE_POSTED_TYPE_CODE", 2)
+
+MESSAGE_REPLY_POINTS = env.int("MESSAGE_REPLY_TYPE_CODE", 2)
+REACTION_POINTS = env.int("MESSAGE_REACTION_TYPE_CODE", 1)
+POSTED_MESSAGE_POINTS = env.int("MESSAGE_POSTED_TYPE_CODE", 1)
```

Comment:
Name of the env variables is duplicate here which should probably not be the case.
Why do we need the type code exposed as env variables again? Can't we hard code them?

Reply:
Good catch! I'll fix it. As for the moment, the reason why we made them environment variables was to easily experiment with how we prioritize different upvote types and what weights should we give them, without having to push to GitHub all the time

Reply:
But we could hardcore the type but at the same time leaving the weights as an env variable

Reply:
Done!